spec/abci/abci++_basic_concepts.md

```diff
@@ -6,7 +6,7 @@ title: Overview and basic concepts
 ## Outline
 
 - [Overview and basic concepts](#overview-and-basic-concepts)
-    - [ABCI++ vs. ABCI](#abci-vs-abci)
+    - [ABCI 2.0 vs. ABCI](#abci-vs-abci)
```

Comment:
Maybe "vs. legacy ABCI"?

Reply:
The link is also broken, missing the 2.0 part.

Reply:
fixed